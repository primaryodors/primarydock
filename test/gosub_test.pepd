
ECHO "Script begin."

GOSUB subroutine
ECHO "Back from subroutine."
GOSUB subroutine2
ECHO "Back from subroutine2."

EXIT

subroutine:
ECHO "Subroutine called."
RETURN

subroutine2:
ECHO "Subroutine2 called."
GOSUB subroutine3
ECHO "Back from subroutine3."
RETURN

subroutine3:
ECHO "Subroutine3 called."
RETURN

