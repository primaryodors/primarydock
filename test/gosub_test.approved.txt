Script begin.
Subroutine called.
Back from subroutine.
Subroutine2 called.
Subroutine3 called.
Back from subroutine3.
Back from subroutine2.
